src/movement/thruster_sabertooth.cpp

```diff
@@ -117,5 +149,12 @@ int main(int argc, char **argv)
 
   ros::spin();
 
+  uint8_t serial_data[4];
```

Comment:
@bkallaher, this code here sends the stop signal to all the thrusters when the node shuts down.